clone_linkedin/post/views.py

```diff
@@ -53,3 +57,32 @@ def update(self, request, pk=None):
     def destroy(self, request, pk=None):
         self.get_object().delete()
         return Response()
+        
+    # GET, POST, PUT, DELETE /posts/:id/reaction/
+    @action(detail=True, methods=['GET', 'POST', 'PUT', 'DELETE'], url_path='reaction')
+    def reaction(self, request, pk=None):
+        post = self.get_object()
+        data = request.data.copy()
+        if self.request.method == 'GET':
+            reaction = self.get_object().postReactions.all()
```

Comment:
사소한 거긴 하지만... `self.get_object()` 또 안 하고 그냥 `post` 변수 사용해도 될 것 같아요
```suggestion
            reaction = post.postReactions.all()
```